ui/src/authors/containers/__tests__/CitationSummaryContainer.test.jsx

```diff
@@ -0,0 +1,67 @@
+import React from 'react';
+import { shallow, mount } from 'enzyme';
+import { fromJS } from 'immutable';
+
+import { getStoreWithState, getStore } from '../../../fixtures/store';
+import { fetchCitationSummary } from '../../../actions/citations';
+import CitationSummaryContainer from '../CitationSummaryContainer';
+
+jest.mock('../../../actions/citations');
+
+describe('CitationSummaryContainer', () => {
```

Comment:
Could you add the case it fetches when the searchQuery is updated, and one when it's updated but with the same value.